customer-churn/kubeflow_requirements.txt

```diff
@@ -1,9 +1,8 @@
 feature_engine==1.2.0
```

Comment:
There is no difference I can find between this file and the `requirements.txt` file. Can you maybe remove this, so we only have one ? Make sure to update the `seldon_deployment_pipeline.py` and `training_pipelines.py` files to point to `requirements.txt` instead.

Reply:
removed this file